### DISCUSSION TITLE
Solved

### DISCUSSION THREAD

**Discussion by oknchm** [spencerwooo/onedrive-vercel-index#854] [Q&A]
Problem has been solved but I can't delete my post.

~~I'm on this part of setting up the index (https://ovi.swo.moe/docs/getting-started#create-a-redis-connection) and generated a `REDIS_URL` using Upstash but it did not automatically appear on Environment Variables like said here:~~

~~> (ignore the final step Create Your Redis Client, that is already taken care of in onedrive-vercel-index)~~

~~So I added the `REDIS_URL` manually on Environment Variables, redeployed on Vercel, but I get `500 Internal Server Error`.~~

~~Hope you can guide me, thank you.~~

**Comment by spencerwooo** [comment]
Never mind, you can just leave it here. :) Glad the problem is solved.
